### ISSUE TITLE
When declaring class attributes outside the __init__ method, comment on each logical block:  

### ISSUE THREAD

**Issue by jonatasbaldin** [jonatasbaldin/coding-style#3]
As I was writing some docstrings (using vim) this method started to be a pain to consult the docstrings with from another file. I could read the class description but not its arguments!

I guess we could add docstrings attributes for **Abstract models** only. Since its abstracted, it should not have too much different logical blocks separation. 

```python
# Good
class Foo(SomeAbstractModel):
    """
    Class description.
    
    Attributes:
    <string> first_name -- User first name.
    <int> age -- Users age.
    """

    first_name = 'Bob'
    age = 30

# Bad
class Foo(SomeAbstractModel):
    """
    Class description.
    """

    # User information
    first_name = 'Bob'
    age = 30

    # User social media
    twitter_profile = '@bob'
    facebook_profile = 'facebook.com/bob'
```

**Comment by jonatasbaldin** [comment]
@smaniotto

**Comment by smaniotto** [comment]
The point of using on abstract models only is because of inheritance? That makes sense, although non-abstract models can also be inherited.

Doing some research I found this: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

I think they have this approach that is backend by a standard:
```python
class MyClass(object):
    """ This is MyClass, it's just a random collection of words to describe
          something that does not exist.
    """
    attribute1 = 'value 1'
    """ Defines the value for the first attribute lalala """
    attribute2 = 2
    """ Defines the value for the second attribute lelele """
```

Or even with more spacing:
```python
class MyClass(object):
    """ This is MyClass, it's just a random collection of words to describe
          something that does not exist.
    """

    attribute1 = 'value 1'
    """ Defines the value for the first attribute lalala """

    attribute2 = 2
    """ Defines the value for the second attribute lelele """
```

This way we won't have gigantic docstrings before each class attributes and we could add documentation to all classes.

**Comment by smaniotto** [comment]
+ it's probably interpreted by VIM as documentation. (Could you double-check?)

**Comment by jonatasbaldin** [comment]
When using abstracted classes, Django only creates one table to the model class that extends it with all its attributes, so it has fields that are not explicit described in the class.

Said that, I guess we should use docstrings, because its easier to get information from the child class.

*Also, vim and other text editors does not interpret this way of documentation* :(

**Comment by jonatasbaldin** [comment]
Actually I see your point, it can grow really fast if the model has lots of fields.

Or we make my adjustment and have enormous docstrings or we need to jump in and out the classes definition to see its attributes.

**Comment by smaniotto** [comment]
Yep, that's the point. Isn't the *under-the-attribute-docstring* version counted as docstring for VIM?

**Comment by jonatasbaldin** [comment]
Nope :(

But I think we should keep class attributes outside the docstring, as you said. Too much duplicated information.
